dbms/src/Storages/Page/V3/PageDirectory.h

```diff
@@ -70,6 +70,11 @@ class PageDirectorySnapshot : public DB::PageStorageSnapshot
         return create_thread;
     }
 
+    String getTracingId() const
```

Comment:
I think not adding the function `getTracingId` and `getTid`, simply use `tracing_id` and `create_thread` directly is also ok.
Cause they are const and won't be changed once the snapshot is created.

After all, adding or not adding are both ok to me.